plugins/CoreVisualizations/angularjs/series-picker/series-picker.component.html

```diff
@@ -0,0 +1,49 @@
+<div
+    class="jqplot-seriespicker"
+    ng-class="{open: $ctrl.isPopupVisible}"
+    ng-mouseenter="$ctrl.isPopupVisible = true"
```

Comment:
@diosmosis just wondering... with components... can we as well use a syntax like `controllerAs` and refer to their controller name specifically to make sure we work in the right scope? Or do components not have this problem?

Reply:
You can supply `controllerAs`, however the default for components is `$ctrl`: https://docs.angularjs.org/guide/component#comparison-between-directive-definition-and-component-definition

So you don't have to explicitly supply it. When coding this, I thought it was better not to have different controller variables all over the place (ie, instead of `periodSelector` in one component, `seriesPicker` in another, `dataTable` in another, everything uses `$ctrl`). Seems more consistent.

Reply:
Sweet 👍